apiserver/facades/schema.json

```diff
@@ -7004,30 +7004,6 @@
                     },
                     "additionalProperties": false
                 },
-                "CharmDeployment": {
-                    "type": "object",
-                    "properties": {
-                        "min-version": {
-                            "type": "string"
-                        },
-                        "mode": {
-                            "type": "string"
-                        },
-                        "service": {
-                            "type": "string"
-                        },
-                        "type": {
-                            "type": "string"
-                        }
-                    },
-                    "additionalProperties": false,
-                    "required": [
-                        "type",
-                        "mode",
-                        "service",
-                        "min-version"
-                    ]
-                },
```

Comment:
You need to bump the facade.

Reply:
As discussed, this will be done in another PR